chia/rpc/wallet_rpc_api.py

```diff
@@ -1727,6 +1729,30 @@ async def nft_set_nft_did(self, request):
         spend_bundle = await nft_wallet.set_nft_did(nft_coin_info, did_id, fee=fee)
         return {"wallet_id": wallet_id, "success": True, "spend_bundle": spend_bundle}
 
+    async def nft_set_bulk_nft_did(self, request):
+        if len(request["nft_coin_list"]) > MAX_NFT_CHUNK_SIZE:
+            return {"success": False, "error": f"You can only set {MAX_NFT_CHUNK_SIZE} NFTs at once"}
+        wallet_id = uint32(request["wallet_id"])
+        nft_wallet = self.service.wallet_state_manager.wallets[wallet_id]
+        assert isinstance(nft_wallet, NFTWallet)
+        did_id = request.get("did_id", "")
```

Comment:
you can set the default value to `b""` here so you don't have to set it in the if